easybuild/easyconfigs/s/SoX/SoX-14.4.2-GCCcore-11.3.0.eb

```
@@ -0,0 +1,36 @@
+# Institution: IT4Innovations National Supercomputing Center, Czech Republic
+# Author: Jakub Kropacek
+# License: GPLv3
+# Year: 2022
+
+easyblock = 'ConfigureMake'
+
+name = 'SoX'
+version = '14.4.2'
+
+homepage = 'http://sox.sourceforge.net/'
+docurls = 'http://sox.sourceforge.net/Docs/Documentation'
+description = """Sound eXchange, the Swiss Army knife of audio manipulation"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ['https://sourceforge.net/projects/sox/files/sox/%(version)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['b45f598643ffbd8e363ff24d61166ccec4836fea6d3888881b8df53e3bb55f6c']
+
+builddependencies = [('binutils', '2.38')]
+
+dependencies = [
+    ('FLAC', '1.3.4'),
+    ('LAME', '3.100'),
+    ('libvorbis', '1.3.7'),
+    ('FFmpeg', '5.0.1'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/play', 'bin/rec', 'bin/sox', 'bin/soxi', 'include/sox.h',
+              'lib/libsox.la', 'lib/libsox.a', 'lib/pkgconfig/sox.pc', 'lib/libsox.%s' % SHLIB_EXT],
+    'dirs': ['bin', 'include', 'lib', 'lib/pkgconfig', 'share/man'],
+}
+
```

One other small improvement that could be made (which we started doing recently for new easyconfigs), is adding a simple sanity check command to ensure the binary can actually be run (and to detect, for instance, missing libraries). Usually something like `app --help` is already sufficient, which also seems to work for `sox`:
```suggestion

sanity_check_commands = ['sox --help']

```

(feel free to add more/others, as I'm not really familiar with `SoX`)

---

I'm no SoX expert, but I guess we would need actual sound files to test `sox` functionality besides `--help` and `--version`.